src/source/Crypto/Dtls_openssl.c

```diff
@@ -539,6 +541,7 @@ STATUS dtlsSessionHandshakeInThread(PDtlsSession pDtlsSession, BOOL isServer)
                         }
                         CHK_STATUS(dtlsCheckOutgoingDataBuffer(pDtlsSession));
```

Comment:
Let us remove this since we are checking this 2 lines down anyways.

Reply:
I think the one that was added was added incorrectly from a merge. Should I instead remove the new one since this bracket is inside an error check?